src/common/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode } from "react";
+import classNames from "classnames/bind";
+import YesNoModal from "@/common/components/Modal/YesNoModal/YesNoModal";
+import ConfirmModal from "@/common/components/Modal/ConfirmModal/ConfirmModal";
+
+import styles from "@/common/components/Modal/Modal.module.scss";
+import Portal from "./ModalPortal";
+import WarnModal from "./WarnModal/WarnModal";
+
+const cn = classNames.bind(styles);
+
+type Props = {
+  children: ReactNode;
+};
+
+export default function Modal({ children }: Props) {
+  return (
+    <Portal>
+      <div className={cn("background")}>{children}</div>
```

Comment:
보통 이 단어로 표현해주는 편입니다!
```suggestion
      <div className={cn("dimmed")}>{children}</div>
```